src/docs/platform-automation/pruning.md

```diff
@@ -24,17 +24,18 @@ Last updated: **March 8, 2024**
 There are a few CronJobs set up on the OpenShift clusters to help prune old objects and keep the cluster clean. This page discusses each of the different object types and how and when they are pruned.
 
 ## On this page
-* **[DeploymentConfigs](#deploymentconfigs)**
+* **[Deployments](#deployments)**
 * **[Builds](#builds)**
 * **[ImageStreams](#imagestreams)**
 * **[Pipelines](#pipelines)**
 * **[Related pages](#related-pages)**
 
-## DeploymentConfigs
+## Deployments
 
-When a `DeploymentConfig` rolls out a new version, it creates a new `ReplicationController`. Over time, the accumulation of these objects can happen. Therefore, older ones are routinely removed on a daily basis through pruning as well.
+When a `Deployment` rolls out a new version, it creates a new `ReplicaSet`. Over time, these ReplicaSets can accumulate as each new deployment creates a new one. OpenShift automatically manages the history of ReplicaSets by retaining a configurable number of old ReplicaSets and scaling them down to 0 replicas. By default, older ReplicaSets are not removed but are instead kept in a scaled-down state. However, you can configure a retention policy to limit the number of old ReplicaSets retained, or manually prune them if needed.
 
 A few settings are passed to the pruner job:
+<!-- Steven please update this part: -->
```

Comment:
https://docs.openshift.com/container-platform/4.14/applications/pruning-objects.html#pruning-deployments_pruning-objects

> To also prune replica sets associated with Deployment objects, use the --replica-sets flag. **This flag is currently a Technology Preview feature.**

Since its tech preview we haven't turned that on yet. So only DCs are getting pruned, not Deploys

Reply:
kk reverted the changes on this part!